k2mosaic/mosaic.py

```diff
@@ -171,6 +182,12 @@ def _make_primary_hdu(self):
         hdu.header.cards['DATE-OBS'].comment = 'TSTART as UTC calendar date'
         hdu.header['DATE-END'] = self.dateend
         hdu.header.cards['DATE-END'].comment = 'TSTOP as UTC calendar date'
+        hdu.header['MJD-BEG'] = self.mjdbeg
+        hdu.header.cards['MJD-BEG'].comment = 'TSTART as modified barycentric Julian date'
+        hdu.header['MJD-end'] = self.mjdend
```

Comment:
This probably doesn't change the output, let's use uppercase here:
```suggestion
        hdu.header['MJD-END'] = self.mjdend
```